lib/modules/manager/bazel/index.ts

```diff
@@ -9,7 +9,11 @@ import { extractPackageFile } from './extract';
 export { extractPackageFile, updateArtifacts };
 
 export const defaultConfig = {
-  fileMatch: ['(^|/)WORKSPACE(|\\.bazel|\\.bzlmod)$', '\\.bzl$'],
+  fileMatch: [
+    '(^|/)WORKSPACE(\\.bzlmod)?$',
+    '\\.WORKSPACE\\.bazel$',
+    '\\.bzl$',
+  ],
```

Comment:
The earlier pattern allowed files named `WORKSPACE.bazel`, while the new one does not. Is that intended? You seem to have both widened and narrowed the filters at the same time

Reply:
Thanks for pointing this out, I have updated as suggested.